js/block.js

```diff
@@ -3134,7 +3134,7 @@ function Block(protoblock, blocks, overrideName) {
                     this._piemenuNumber([-25, -20, -15, -10, -5, 0, 5, 10, 15, 20, 25], this.value);
                     break;
                 case 'hertz':
-                    this._piemenuNumber([220, 261, 293, 329, 349, 392, 440, 493, 523, 587, 659, 698,783,880], this.value);
+                    this._piemenuNumber([220, 247, 262, 294, 330, 355, 392, 440, 494, 530, 590, 660, 700,800,880], this.value);
```

Comment:
Can you please explain this change?

Reply:
I've just adjusted the frequency because when I converted it to pitch, it gives out wrong pitch.

Reply:
I changed it to:
this._piemenuNumber([220, 247, 262, 294, 330, 349, 392, 440, 494, 523, 587, 659, 698, 784, 880], this.value);

Reply:
Sure, can be changed as well, but I'm just concerned that it may be converting the pitch with flat (which should not be). I test whether it's a flat or not by printing the actualPitch variable under the hertzPreview.